packages/discovery-provider/plugins/pedalboard/apps/trending-challenge-rewards/src/app.ts

```diff
@@ -137,8 +137,13 @@ const canSuccessfullyAttest = async (
     challengeId,
     oracleEthAddress
   )
-  const res = await fetch(url)
-  return res && res.ok
+  try {
+      const res = await fetch(url)
```

Comment:
indentation is weird here just fyi

Reply:
should it be more indented? I'm not sure what's weird here